oarepo_oaipmh_harvester/ui/oai_run/templates/semantic-ui/oai_run_ui/RunMain.jinja

```diff
@@ -1,78 +1,71 @@
-{#def metadata, ui, layout #}
+{#def metadata, ui #}
+<h1 style="margin-bottom: 1em">{{ metadata.title or metadata.started }}</h1>
+<dl class="ui very basic table">
 
-<div class="ui grid">
-    <div class="eight wide column">
+    <dt>{{ _('harvester.label') }}</dt>
+    <dd>
+        <a href="/oai/harvest/harvesters/{{ metadata.harvester.id }}">&laquo; {{ metadata.harvester.name }}</a>
+    </dd>
 
-        <dl class="ui very basic table">
-            <h1 style="margin-bottom: 1em">{{ metadata.title or metadata.started }}</h1>
+    <dt>{{ _('status.label') }}</dt>
+    <dd>
+        <Status>{{ metadata.status[-1] }}</Status>
+    </dd>
 
-            <dt>{{ _('harvester.label') }}</dt>
-            <dd>
-                <a href="/oai/harvest/harvesters/{{ metadata.harvester.id }}">&laquo; {{ metadata.harvester.name }}</a>
-            </dd>
+    <Field label={ _('created.label') }> {{ metadata.created }}</Field>
+    <Field label={ _('updated.label') }> {{ metadata.updated }}</Field>
 
-            <dt>{{ _('status.label') }}</dt>
-            <dd>
-                <Status>{{ metadata.status[-1] }}</Status>
-            </dd>
+    <dt>{{ _('started.label') }}</dt>
+    <dd>
+        {{ metadata.started }}
+    </dd>
+    {% if metadata.finished %}
+        <dt>{{ _('finished.label') }}</dt>
+        <dd>
+            {{ metadata.finished }}
+        </dd>
+    {% endif %}
+    {% if metadata.duration %}
+        <dt>{{ _('duration.label') }}</dt>
+        <dd>
+            {{ metadata.duration }}
+        </dd>
+    {% endif %}
 
-            <Field label={ _('created.label') }> {{ metadata.created }}</Field>
-            <Field label={ _('updated.label') }> {{ metadata.updated }}</Field>
+    <Field label={ _('id.label' ) }> {{ metadata.id }}</Field>
+</dl>
 
-            <dt>{{ _('started.label') }}</dt>
-            <dd>
-                {{ metadata.started }}
-            </dd>
-            {% if metadata.finished %}
-                <dt>{{ _('finished.label') }}</dt>
-                <dd>
-                    {{ metadata.finished }}
-                </dd>
-            {% endif %}
-            {% if metadata.duration %}
-                <dt>{{ _('duration.label') }}</dt>
-                <dd>
-                    {{ metadata.duration }}
-                </dd>
-            {% endif %}
 
-            <Field label={ _('id.label' ) }> {{ metadata.id }}</Field>
-        </dl>
-    </div>
-    <div class="eight wide column ui segment">
+<dl class="ui very basic table">
 
-        <dl class="ui very basic table">
+    <dt>{{ _('batches_links.label') }}</dt>
+    <dd>
+        <a href="/oai/harvest/batches/?q=&f=run_id%3A{{ metadata.id }}&l=list&p=1&s=10&sort=newest">&raquo; {{ _('batches_link.label') }}</a>
```

Comment:
Rather than hardcoding URL, isn't the correct url endpoint in props? (ui links?)

Reply:
@mirekys Ahoj, it is problematic, because one detail, is referencing something from another detail or is this just wrong i nthe setup? Meaning if I am referencing batches, in RunMain jinja, runMain "record" does not contain ui links for batches.

Reply:
@Ducica I see, looks like a potential future improvement on BE side, to include links to related items, lets not tackle it in this PR then